Examples/Algorithms/TrackFindingExaTrkX/src/PrototracksToParameters.cpp

```diff
@@ -59,17 +59,22 @@ ProcessCode PrototracksToParameters::execute(
   const auto &sps = m_inputSpacePoints(ctx);
   auto prototracks = m_inputProtoTracks(ctx);
 
-  // Make some lookup tables. Allocate space for the maximum number of indices
-  // (max 2 source links per spacepoint)
+  // Make some lookup tables and pre-allocate some space
+  // Note this is a heuristic, since it is not garantueed that each measurement
+  // is part of a spacepoint
```

Comment:
alternatively you could wire in the measurement container which might be more efficient that resizing the vector later

or using a map / hash table?

Reply:
Yeah you're right, but I don't want to change the interface right now...

Reply:
I suppose we would need some insert performance and some lookup performance. if `map` or `unordered_map` is faster in this case I cannot foresee

Reply:
I've seen the out of bounds access in 1 of 500 single muon events, and never in ttbar, so I think this fine...